lib/serre.gi

```diff
@@ -352,6 +352,11 @@ IrreducibleDecompositionCollectedHybrid@ := function(rho)
                                                                  IrrepCanonicalSummand@(rho,
                                                                                         irrep));
     parallel := ValueOption("parallel");
+    if IsBoundGlobal("ParListByFork") then
+      ParListByFork := ValueGlobal("ParListByFork");
+    else
+      parallel := fail;
```

Comment:
If `io` is not loaded and `parallel <> fail` then this will run into an error a few lines below when it tries to invoke `ParListByFork`. I suggest we instead show the same error here as in the other function:
```suggestion
    elif parallel <> fail then
        Error("I need the package IO >= 4.7.0 to use the parallel option!");
```